app/routers/user_exercise.py

```diff
@@ -0,0 +1,50 @@
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.orm import Session
+from datetime import datetime
+from app.database.models import User, UserExercise
+from app.internal.utils import save
+
+
+def create_user_exercise(user: User, session: Session) -> UserExercise:
+    """
+    Create and save new user exercise
+    """
+    if not does_user_exercise_exist(session=session, userid=user.id):
+        user_exercise = UserExercise(
+            user_id=user.id,
+            start_date=datetime.now()
+            )
+        save(user_exercise, session=session)
+    else:
+        user_exercise = update_user_exercise(user, session)
+    return user_exercise
+
+
+def does_user_exercise_exist(session: Session, userid: int) -> bool:
+    """
+      Checking if user exercise with user id is exist
```

Comment:
Consider:  Check if a user exercise for user id exists.

Reply:
fix the issue